Foundation/NSOperation.swift

```diff
@@ -47,7 +47,6 @@ open class Operation : NSObject {
 #endif
     }
     
-    /// - Note: Operations that are asynchronous from the execution of the operation queue itself are not supported since there is no KVO to trigger the finish.
     open func start() {
```

Comment:
@phausler @parkera I've added a test for this. Please could you review and run CI?